compiler/crates/relay-compiler/Cargo.toml

```diff
@@ -14,6 +14,10 @@ path = "src/lib.rs"
 name = "relay_compiler_compile_relay_artifacts_test"
 path = "tests/compile_relay_artifacts_test.rs"
 
+[features]
+default = []
```

Comment:
default is magic, and gets applied—by default. I was thinking of first going `is_OSS` and `is_FB`, but that _could_ be dangerous because what-if, both are applied.

The array simply allows us to nest features, ie maybe have a feature for `fb_connections`, that the `is_FB` turns on. etc..